src/Microsoft.DocAsCode.MarkdigEngine.Extensions/CodeSnippet/HtmlCodeSnippetRenderer.cs

```diff
@@ -181,14 +181,11 @@ protected override void Write(HtmlRenderer renderer, CodeSnippet codeSnippet)
                 return;
             }
 
-            using (InclusionContext.PushFile(codeSnippetPath))
-            {
-                codeSnippet.SetAttributeString();
+            codeSnippet.SetAttributeString();
```

Comment:
Why is code snippet not pushed any more?

Reply:
I think it was added by accident, this is only intended for `[!include()]`